superdesk/default_settings.py

```diff
@@ -598,4 +599,4 @@ def celery_queue(name):
 #: The number of hours before temporary media files are purged
 TEMP_FILE_EXPIRY_HOURS = int(env('TEMP_FILE_EXPIRY_HOURS', 24))
 #: The number of days before content api items are removed, defaults to approximately 6 months
```

Comment:
Please changes the comment too in this case...

@petrjasek is it ok a such default?